### PR TITLE
Add ResNet50 Inference to CI

### PR DESCRIPTION
**Pull request by bani-intelaipg**

### CODE REVIEW

File: test/ci/buildkite/test_runner.py [hunk 1]
```diff
@@ -125,6 +130,14 @@ def main():
                     iterations = 100
             run_resnet50_from_artifacts('./', arguments.artifacts_dir,
                                         batch_size, iterations)
+    elif (arguments.test_resnet50_infer):
+        if get_os_type() == 'Darwin':
+            sys.exit(1)
```
Comment (adk9) [inline test/ci/buildkite/test_runner.py:135]: You probably don't need this but, at the very least, print a warning ("RN50 inference test not supported on Darwin") before exiting.

**Comment by adk9** [comment]
@googlebot I consent.

File: test/ci/buildkite/test_runner.py (hunk 1 above)
Comment (bani-intelaipg) [reply test/ci/buildkite/test_runner.py]: Ok

File: tools/test_utils.py [hunk 2]
```diff
@@ -575,6 +575,162 @@ def run_resnet50_forward_pass_from_artifacts(ngraph_tf_src_dir, artifact_dir,
...
+                            pretrained_model)
+    else:
+        print("Using existing pretrained model file: " + pretrained_model)
+
+    os.chdir(root_pwd)
+    cmd = [
+        'python',
+        ngraph_tf_src_dir + '/test/python/test_rn50_infer.py',
+        '--input-graph',
+        pretrained_model,
```
Comment (adk9) [inline tools/test_utils.py:719]: Pass `batch_size` to the script here. The script doesn't take `iterations` but takes the number of (synthetic) images to process.

File: tools/test_utils.py [hunk 3]
```diff
@@ -575,6 +575,162 @@ def run_resnet50_forward_pass_from_artifacts(ngraph_tf_src_dir, artifact_dir,
     os.chdir(root_pwd)
 
 
+# See https://github.com/tensorflow/benchmarks/tree/master/perfzero
+def run_tfperfzero_resnet50_infer_from_artifacts(
```
Comment (adk9) [inline tools/test_utils.py:579]: What is this for?

Comment (bani-intelaipg) [reply tools/test_utils.py:579]: For future extension to use TF recommended benchmarking

Comment (adk9) [reply tools/test_utils.py:579]: Maybe add this when we actually get around to testing/benchmarking with PerfZero?

**Pull request merged**